engine-dmn/engine/src/main/java/org/camunda/bpm/dmn/engine/impl/DefaultDmnEngineConfiguration.java

```diff
@@ -521,4 +566,31 @@ public DefaultDmnEngineConfiguration setFeelCustomFunctionProviders(List<FeelCus
     return this;
   }
 
+  /**
+   * @return whether FEEL legacy behavior is enabled or not
+   */
+  public boolean isEnableFeelLegacyBehavior() {
+    return enableFeelLegacyBehavior;
+  }
+
+  /**
+   * Controls the whether the FEEL legacy behavior is enabled or not
+   *
+   * @param enableFeelLegacyBehavior the FEEL legacy behavior
+   */
+  public void setEnableFeelLegacyBehavior(boolean enableFeelLegacyBehavior) {
+    this.enableFeelLegacyBehavior = enableFeelLegacyBehavior;
+  }
+
+  /**
+   * Controls the whether the FEEL legacy behavior is enabled or not
+   *
+   * @param enableFeelLegacyBehavior the FEEL legacy behavior
+   * @return this
+   */
+  public DefaultDmnEngineConfiguration enableFeelLegacyBehavior(boolean enableFeelLegacyBehavior) {
+    this.enableFeelLegacyBehavior = enableFeelLegacyBehavior;
```

Comment:
I think we should use the `setEnableFeelLegacyBehavior` here so that we have a single point where the flag is changed. This might lead to inconsistent behaviour if a user isn't carefull.

Reply:
👍